test/authentication_tests.coffee

```diff
@@ -50,7 +50,9 @@ Meteor.startup ->
 
         next()
 
-    it 'should not allow a user with wrong password to login', (test, next) ->
+    it 'should not allow a user with wrong password to login and should respond after 500 msec', (test, next) ->
```

Comment:
Sounds like solid reasoning. I would just move the comment below right above this and change it to:
```coffeescript
# This test should take 500 msec or more. To speed up testing, these two tests have been combined.
```
Don't use "I" in any comments. It doesn't help anyone reading it, since they have no idea who "I" is.